mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1046,8 +1046,12 @@ defvar LdsBufferTypes = GemmInputTypes # [VectorOfRankAndType<[1], GemmInputType
 def Rock_BlockwiseGemmOp:
     Rock_Op<"blockwise_gemm">,
     Arguments<(ins MemRefRankOf<GemmInputTypes, [3]>:$matrixA,
+                   BoolAttr : $isKContiguousDimA,
                    MemRefRankOf<GemmInputTypes, [3]>:$matrixB,
+                   BoolAttr : $isKContiguousDimB,
                    MemRefRankOf<GemmOutputTypes, [2]>:$matrixC,
+                   I32Attr:$copyMPerThread,
```

Comment:
nit : we could drop the copy* prefix as it doesn;t need to be copied -- it could also be computed.

Reply:
The idea here is that the `dPerThread` is how many elements each thread copies from global to LDS, which can be different from how many elements each thread computes. Happy to proceed if you want to still change the name, I just wanted to give you the context of why I picked it

Reply:
That dPerThread is computed not copied.